src/dev/src/app/tabs/tabs.demo.module.ts

```diff
@@ -12,7 +12,7 @@ import { TabsAngularDemo } from './tabs-angular';
 import { TabsStaticDemo } from './tabs-static';
 import { TabsDemo } from './tabs.demo';
 import { ROUTING } from './tabs.demo.routing';
-import { ClrKeyFocusModule } from 'src/clr-angular/utils/focus/key-focus/key-focus.module';
+import { ClrKeyFocusModule } from './../../../../clr-angular/utils/focus/key-focus/key-focus.module';
```

Comment:
We can actually import from `@clr/angular` since this is the dev app.

Reply:
It is not a public directive so its not exposed through `@clr/angular`